modules/javafx.graphics/src/main/docs/javafx/scene/doc-files/cssref.html

```diff
@@ -362,7 +364,7 @@ <h2>Contents</h2>
           </li>
         </ul>
       </li>
-      <li><a href="#controls">Controls</a>
+      <li><a id="controls">Controls</a>
```

Comment:
We should consider leaving this as `href` and adding a `Controls` heading instead.
It should be here: 
![image](https://github.com/openjdk/jfx/assets/66004280/be1134e5-91df-4650-b56d-3f6ecd7cc18a)
As it is for all other headers e.g. `Nodes`
![image](https://github.com/openjdk/jfx/assets/66004280/17eba5ee-9804-4452-aaf2-b8448a9d4cb4)

Reply:
you are right

Reply:
You need to add a `Controls` headline still.
`<h2><a id="nodes">Nodes</a></h2>`
It should be above the `javafx.scene.control` row.
Then the `href` will work and is consistent with the other headers.

![image](https://github.com/openjdk/jfx/assets/66004280/fd80afd3-5b12-401d-a9f2-d14f123927f6)

Reply:
@Maran23 I see what you are saying now, and you are right. The change to the link should be reverted, and a new heading added as you suggest.